bin/lint

```diff
@@ -1,4 +1,12 @@
 #!/bin/sh
 
 set -e
-bundle exec rubocop
+
+if [ $# -eq 1 ] && [ $1 = "--pre-commit" ]; then
+  echo "Running Rubocop..."
+  bundle exec rubocop -a > /dev/null
```

Comment:
Like this you can directly commit your violations after trying to commit a bad commit